Userland/Libraries/LibSQL/AST/Expression.cpp

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <LibELF/DynamicObject.h>
```

Comment:
What's this `#include` for? Seems unused

Reply:
I have no idea. Probably some CLion autocomplete artifact.

Attention to detail. I need to have it.